source/ui.js

```diff
@@ -170,6 +170,25 @@ module.exports = async (options, pkg) => {
 		}
 	}
 
+	if (options.unknown) {
+		const answers = await inquirer.prompt([
+			{
```

Comment:
Could you change this to conform with the way the rest of the prompts are written (i.e. put this in the same line as the one above)?

Reply:
Done. 👌